CHANGELOG.md

```diff
@@ -10,6 +10,7 @@
 * added a new command **run-test-playbook** to run a test playbook in a given XSOAR instance.
 * Fixed an issue in **format** when running on a modified YML, that the `id` value is not changed to its old `id` value.
 * Enhancement for **split** command, replace `ApiModule` code block to `import` when splitting a YML.
+* Fixed an issue in the **zip-packs** command, it will now add new indicator types to the zipped pack.
```

Comment:
Fixed an issue where the indicator type missing from the Pack Content when uploading using **zip-packs** to zip the Pack.
Something like this.